### ISSUE TITLE
HLL flux FPEs when all speeds are zero

### ISSUE THREAD

**Issue by fmahebert** [sxs-collaboration/spectre#1841]
# Bug reports:

### Expected behavior:

HLL flux works generally.

### Current behavior:

When all speeds are 0 the HLL computation FPE's, presumably on this line (`Hll.hpp:176`):
```c++
const DataVector one_over_cp_minus_cm =
    1.0 / (get(max_signal_speed) - get(min_signal_speed));
```

However, because the HLL flux also has factors of the speeds in the numerator, the 0-speed case should be handled to return (presumably) 0, rather than FPE.

For a concrete example of when this arises: when evolving the NewtonianEuler system in a vacuum region where density = velocity = energy = 0, and all characteristic speeds are 0.

**Comment by nilsdeppe** [comment]
Isn't the more fundamental issue here that the hydro approximation isn't valid in vacuum? You assume something like Avogadro's number of particles, which is just not true in vacuum.

**Comment by fmahebert** [comment]
I agree about the fundamental issue. Still, it might be nice for the code to fail more gracefully than by FPE...
